website/client/js/services/statServices.js

```diff
@@ -131,6 +131,6 @@
       mountMasterProgress: mountMasterProgress,
       mpDisplay: mpDisplay,
       totalCount: totalCount
-    }
+    };
```

Comment:
this was a missing semicolon from previous versions of the file, changed to improve consistency and code climate rating